go.mod

```diff
@@ -265,9 +265,6 @@ replace (
 	// latest tag resolves to a very old version. this is only used for spinning up local test registries
 	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
 
-	// pin to ginkgo v1 version of gomega (v1.18+ relies on ginkgo v2)
-	github.com/onsi/gomega => github.com/onsi/gomega v1.17.0
-
```

Comment:
This was introduced by DanS [here](https://github.com/openshift/operator-framework-olm/commit/275408b28812e5b9d70bd72acbfffbb66e087b96), that was removed in [4.14](https://github.com/openshift/operator-framework-olm/blob/release-4.14/go.mod). Had to remove it from 4.13 manually to get dependencies to resolve correctly. This was not a part of @grokspawn's commit.